__tests__/buildTranslationFiles.spec.ts

```diff
@@ -1,16 +1,22 @@
 import fs from 'fs-extra';
-import {jest} from '@jest/globals';
+import { jest } from '@jest/globals';
 
-import { buildTranslationFiles } from '../src/keys-builder';
-import { getCurrentTranslation } from '../src/keys-builder/utils/get-current-translation';
 import { resetScopes } from '../src/keys-builder/utils/scope.utils';
 import { messages } from '../src/messages';
 import { Config, FileFormats, Translation } from '../src/types';
-import { resolveProjectBasePath } from '../src/utils/resolve-project-base-path';
-
-jest.mock('../src/utils/resolve-project-base-path');
+import {
+  spyOnConsole,
+  spyOnProcess,
+  spyOnResolveProjectBasePath,
```

Comment:
```suggestion
  mockResolveProjectBasePath,
```
We don't care about the spy, just for mocking purposes

Reply:
Done